src/Utils.jl

```diff
@@ -0,0 +1,100 @@
+module Utils
+
+export add_to_session!, remove_from_session!
+using FromFile
+using Base64
+using SHA
+import Pluto
+
+@from "Settings.jl" using Settings
+
+@from "MoreAnalysis.jl" using MoreAnalysis
+@from "Types.jl" using Types
+
+@from "FileHelpers.jl" import FileHelpers:generate_static_export, generate_html
+@from "Export.jl" import Export: Export, try_fromcache, try_tocache, default_index
+
+showall(xs) = Text(join(string.(xs),"\n"))
+pluto_version = Export.try_get_exact_pluto_version()
+
+myhash = base64encode ∘ sha256
+
+function path_hash(path)
+    myhash(read(path))
+end
+
+
+
+function add_to_session!(server_session, notebook_sessions, path, settings, pluto_options, static_export=false, output_dir=".")
```

Comment:
This should not be in Utils.jl because it is a core function. Maybe this file should not be called Utils but SessionManagement.jl or something?